source/contracts/reporting/Market.sol

```diff
@@ -157,11 +149,11 @@ contract Market is DelegationTarget, Typed, Initializable, Ownable, IMarket {
         reportingWindow.getReputationToken().trustedTransfer(msg.sender, allReportersDisputeBondToken, Reporting.allReportersDisputeBondAmount());
         reportingWindow.getUniverse().fork();
         IReportingWindow _newReportingWindow = getUniverse().getReportingWindowForForkEndTime();
-        updateTentativeWinningPayoutDistributionHash(tentativeWinningPayoutDistributionHash);
         return migrateReportingWindow(_newReportingWindow);
     }
 
     function migrateReportingWindow(IReportingWindow _newReportingWindow) private afterInitialized returns (bool) {
+        updateTentativeWinningPayoutDistributionHash(tentativeWinningPayoutDistributionHash);
```

Comment:
I know you probably did this to address the code size issue, but it makes me sad none the less.

Reply:
Actually, would you mind dropping a comment here to move it back out at some point.  `migrateReportingWindow` really shouldn't have a side effect of updating payouts and I'm worried we'll forget it here.

Reply:
Yea this one made me sad too :[ I'll actually a comment about it so if the refactor effort succeeds we can put it where it belongs